static/oas/v1/rosters/rosters.yml

```diff
@@ -0,0 +1,84 @@
+paths:
+  rosters:
+    post: 
+      tags: 
+        - roster
+      summary: Create Roster  
+      requestBody:
+        description: A JSON object with a valid roster
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/schemas/PostRosterRequest"  
```

Comment:
I believe everywhere where you see #/schemas... you need to change it to #/components/schemas...
This is what I am seeing in the current code.

Reply:
changed